<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pintura</class>
 <widget class="QWidget" name="Pintura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pintura</string>
  </property>
  <widget class="QLabel" name="lCursor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tama√±o del cursor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="rRed">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rojo</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rGreen">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Verde</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBlue">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Azul</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
